src/core/border_router/routing_manager.cpp

```diff
@@ -519,47 +572,36 @@ void RoutingManager::EvaluateNat64Prefix(void)
 {
     OT_ASSERT(mIsRunning);
```

Comment:
Not strictly related to this PR, but I would prefer to move this below the variable declarations.

Reply:
Updated. Thanks!